packages/rules/src/mustSatisfyPeerDependencies.ts

```diff
@@ -146,8 +146,8 @@ function checkSatisfyPeerDependencies(context: Context, opts: Options) {
             file: packageJsonPath,
             message: `[1] Package ${packageName} has conflicting inherited ${peerDependencyName} peer dependencies.`,
             longMessage:
-              `Dependency ${peerRequirement.node.packageJson.name} requires ${peerRequirement.range} and ` +
-              `dependency ${mostStrictPeerRequirement.node.packageJson.name} requires ${mostStrictPeerRequirement.range}.`,
+              `Dependency ${peerRequirement.node.packageJson.name} requires '${peerRequirement.range}' and ` +
```

Comment:
```suggestion
              `Dependency ${peerRequirement.node.packageJson.name} requires '${peerRequirement.range}' but ` +
```

Reply:
oooh, good idea.
I'll commit this locally, since I think I'll need to update a test, too